root/release/edit_relationships.tt

```diff
@@ -1,4 +1,4 @@
-[%- WRAPPER 'layout.tt' full_width=1 title=l('Edit Relationships: {release}', {release => release.name}) -%]
+[%- WRAPPER 'layout.tt' full_width=1 title=l('Edit Relationships: {release}', {release => release.name}) gettext_domains=['attributes'] -%]
```

Comment:
This seems odd though since the entity-creation dialog uses an iframe (so it should be sandboxed and have its own window and set of scripts that it loads?).